### DISCUSSION TITLE
brew bump-cask-pr and --url

### DISCUSSION THREAD

**Discussion by ChristianKraemer** [Homebrew/discussions#4570] [Everyday usage]
### Output of `brew config`

```shell
n/a
```

### Output of `brew doctor`

```shell
n/a
```

### Description of issue

Today a new Nextcloud  (https://github.com/Homebrew/homebrew-cask/blob/HEAD/Casks/nextcloud.rb) version was released.

A simple "brew bump-cask-pr  --version 3.9.0 Nextcloud" fails. I think because of a changed download source from "https://github.com/nextcloud/desktop/releases/download/" to "https://github.com/nextcloud**-releases**/desktop/releases/download/".
In the above mentioned recipe I found 'url "https://github.com/nextcloud/desktop/releases/download/v#{version}/Nextcloud-#{version}.pkg"'. Looked at the fine manual and "--url" seemed the way to deal with a new download URL.

But "brew bump-cask-pr --version 3.9.0 --url "https://github.com/nextcloud-releases/desktop/releases/download/v#{version.major_minor_patch}/Nextcloud-#{version}.pkg" nextcloud" fails with "Error: Invalid usage: `--url` is not valid."

"-v" does not help and the documentation seems very frugal.

Can anyone tell how and what for to use "--url". I remember a similar situation in "cask-repair" times where pressing "e" was my friend. Do I really need the whole Clone_Repo-Edit-Create_PR_By_Hand hammer for such a simple change?

And last but not least: If it's not a simple answer, can anyone care for the Nextcloud update?

**Comment by SMillerDev** [comment]
https://github.com/nextcloud-releases/desktop/releases/download/v3.9.0/Nextcloud-3.9.0.pkg seems to be the URL
